source/Entities/Planets/GreenPlanet.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Annex;
+using Annex.Data.Shared;
+using Game.Scenes;
+
+namespace Game.Entities.Planets
+{
+    public class GreenPlanet : Planet
+    {
+        public GreenPlanet(string spritePath, Vector position) : base(spritePath, position)
+        {
+        }
+
+        public override void OnCollision(CollisionEntity entity)
+        {
+            ServiceProvider.SceneService.LoadScene<GreenScreen>();
```

Comment:
I know this is probably a demo for now, but once we have other entities moving in the scene, it's possible they might collide with the planet too. It'd be good to make a check that the colliding entity is actually the player before loading the scene